### PR TITLE
chore: bump ff, group to 0.13 and update dependent crates

### PR DESCRIPTION
**Pull request by huitseeker**
- updates pasta_curves to 0.5.1, removes unneeded fork,
- updates bellperson to 0.25,

## TODO:
- [x] Merge [dependent PR](https://github.com/supranational/pasta-msm/pull/3) in pasta-msm
- [x] Merge [dependent PR](https://github.com/microsoft/Nova/pull/162) in Nova

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -2,7 +2,7 @@ use std::{
     collections::HashMap,
     env::current_dir,
     fs,
-    path::{Path, PathBuf},
+    path::PathBuf,
```
Comment (nalinbhardwaj) [inline src/lib.rs:5]: This is necessary for the WASM build (see CI), could you add it back

**Review by nalinbhardwaj — COMMENTED** [review]
thanks for the upgrades 🙌

**Comment by nalinbhardwaj** [comment]
(looks like waiting on a Nova cargo release)

**Comment by huitseeker** [comment]
@nalinbhardwaj this should be ready to go

**Review by nalinbhardwaj — APPROVED** [review]
looks great, thanks!

**Pull request merged**